stylesheets/bootstrap.min.css

```diff
@@ -1257,7 +1257,7 @@ hr {
     line-height:1.4
 }
 
-@media (min-width:768px) {
+@media (min-width:1300px) {
```

Comment:
I'm not sure if this change is directly causing this, but have some issues with how the nav behaves when the browser width is changed:

1. The "burger" icon appears way too soon (i.e. when the page is still much wider than it would be on a handheld device). It's purpose is to help touchscreen users, but it will appear for many desktop/laptop users also. Look at this screenshot of my entire 15" screen:

![screen shot 2015-05-04 at 3 28 34 pm](https://cloud.githubusercontent.com/assets/277504/7450006/615be620-f272-11e4-88d8-78ebc4e37ada.png)

2. The "burger" icon sometimes disappears to the right. Again, I think it's to do with the 1300px change but having verified.

![Uploading Screen Shot 2015-05-04 at 3.31.07 PM.png…]()